packages/config/config/devices/0x014a/tstat-tbz500.json

```diff
@@ -0,0 +1,625 @@
+{
+	"manufacturer": "Ecolink",
+	"manufacturerId": "0x014a",
+	"label": "TBH500",
+	"description": "Z-Wave Smart Thermostat",
+	"devices": [
+		{
+			"productType": "0x5442",
+			"productId": "0x5439"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "System Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Standard",
+					"value": 0
+				},
+				{
+					"label": "Heat Pump",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Fan Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Gas (No fan w/Heat)",
+					"value": 0
+				},
+				{
+					"label": "Electric (Fan w/Heat)",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "Change Over Type",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "w/Cool",
+					"value": 0
+				},
+				{
+					"label": "w/Heat",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "4",
+			"label": "2nd Stage Heat Enable",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "5",
+			"label": "Aux Heat Enable",
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"defaultValue": 1
+		},
+		{
+			"#": "6",
+			"label": "2nd Stage Cool Enable",
+			"$import": "~/templates/master_template.json#base_enable_disable"
+		},
+		{
+			"#": "7",
+			"label": "Temperature Unit",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Celcius",
+					"value": 0
+				},
+				{
+					"label": "Fahrenheit",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "8",
+			"label": "Minimum Off Time",
+			"unit": "minutes",
+			"valueSize": 1,
+			"minValue": 5,
+			"maxValue": 9,
+			"defaultValue": 5
+		},
+		{
+			"#": "9",
+			"label": "Minimum Run Time",
+			"unit": "minutes",
+			"valueSize": 1,
+			"minValue": 3,
+			"maxValue": 9,
+			"defaultValue": 3
+		},
+		{
+			"#": "10",
+			"label": "Setpoint H/C Delta",
+			"unit": "°F",
```

Comment:
Is this always °F or depending on the chosen unit?

Reply:
I chose to do this because of the following statement in the manual:

Note all temperature related parameters are in degrees F.

Reply:
Ok so the unit is strictly for the sent reports then?

Reply:
I read this to mean that all values used in temperature-related parameters (like Deltas and setpoints) are assumed to be in deg F, which would be consistent with the units listed on those parameters here. Most of these are writeable params. 

There is a user preference in Setup that I believe this parameter refers to, which is the DISPLAYED unit, whereas internally, it appears everything is F. Operation thus far seems consistent with all of that.